### ISSUE TITLE
.config in .ebextension is not working

### ISSUE THREAD

**Issue by csKangGdcomm** [zodern/mup-aws-beanstalk#76]
Hi,

I want to monitor the memory usage on ec2 at AWS.

I found proper way, so i tried.(https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/mon-scripts.html#mon-scripts-perl_prereq, https://www.otreva.com/blog/monitoring-memory-ram-aws-elasticbeanstalk-ec2-instances-cloudwatch/ )

On seconds sources, I passed IAM thing because I think I already provide  authentication info in mup.js .... 

But I fail to monitor the usage. can you help me?

I add a file on ./.ebextensions/eb-memory-monitor.config . (on root meteor app)

the content of files is below.

``` yaml
container_commands:
  00yum:
    test: '[ "$ROOT_URL" == "https://www.myWebsite.com" ]'
    command: "sudo yum install -y perl-Switch perl-DateTime perl-Sys-Syslog perl-LWP-Protocol-https perl-Digest-SHA.x86_64"
    ignoreErrors: false
  01download:
    test: '[ "$ROOT_URL" == "https://www.myWebsite.com" ]'
    command: "curl https://aws-cloudwatch.s3.amazonaws.com/downloads/CloudWatchMonitoringScripts-1.2.2.zip -O"
    ignoreErrors: false
  02extract:
    test: '[ "$ROOT_URL" == "https://www.myWebsite.com" ]'
    command: "unzip CloudWatchMonitoringScripts-1.2.2.zip"
    ignoreErrors: false
  03rmzip:
    test: '[ "$ROOT_URL" == "https://www.myWebsite.com" ]'
    command: "rm CloudWatchMonitoringScripts-1.2.2.zip"
    ignoreErrors: false
  04cdinto:
    test: '[ "$ROOT_URL" == "https://www.myWebsite.com" ]'
    command: "mv aws-scripts-mon/ /home/ec2-user"
    ignoreErrors: false
  05cron:
    test: '[ "$ROOT_URL" == "https://www.myWebsite.com" ]'
    command: "crontab -l | { cat; echo '*/5 * * * * /home/ec2-user/aws-scripts-mon/mon-put-instance-data.pl --mem-util --mem-used --mem-avail'; } | crontab -"
    ignoreErrors: false
```
I expected that the command will run on the ec2 but there is no logs about them and fail to run these commands.

Thanks in advance.

**Comment by joaocarloscabral** [comment]
You can use a custom npm `@settlin/mup-aws-beanstalk`.
Check this > https://github.com/zodern/mup-aws-beanstalk/pull/72

**Comment by zodern** [comment]
Support for custom `.ebextension` will be available in 0.7 - #142
